ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -45,3 +47,40 @@ def test_tensorflow_det(
         fn_tree="linalg.det",
         input=np.asarray(x, dtype=input_dtype),
     )
+
+
+# slogdet
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_float_dtypes,
```

Comment:
Please update your test to use `ivy_tf.valid_float_dtypes` instead.

Reply:
done